index.qmd

```diff
@@ -83,74 +83,82 @@ This distinction between metawebs and realised webs lead to some further definit
+Perhaps the simplest realising process is the principle that organisms feeding randomly will consume resources in proportion to their abundance [@stephensForagingTheory1986]. On its own, such abundance constrained interactions are not necessarily contingent on there being any compatibility between species [@canardEmergenceStructuralPatterns2012; @momalTreebasedInferenceSpecies2020; @pomeranzInferringPredatorPrey2019]. Here, abundance of different prey species influence the distribution of links in a network [@vazquezUnitingPatternProcess2009] by defining a preference linked to individuals among species meeting [@poisotSpeciesWhyEcological2015; @banvilleDecipheringProbabilisticSpecies2024]. Abundance data, linked to a derived metaweb delivers a foundation ruleset that can define the distribution and strength of links - a formal network topology/structure.
 
-**Profitability**
+**Profitability (We call it diet choice above... might be less contentious than profitability)**
 
-It is well established that consumers make more active decisions than eating items in proportion to their abundance [@stephensForagingTheory1986]. Ultimately, consumer choice is underpinned by an energetic cost-benefit framework centred around profitability and defined by traits associated with finding, catching, killing, and consuming a resource [@woottonModularTheoryTrophic2023]. Although energetic constraints can be invoked in a myriad of ways [*e.g.,* @pawarDimensionalityConsumerSearch2012; @portalierMechanicsPredatorPrey2019; @cherifEnvironmentRescueCan2024] we select profitability as a term to capture rules linked to optimal foraging [@pykeOptimalForagingTheory1984] and metabolic theory [@brownMetabolicTheoryEcology2004]; it is a sensible ‘umbrella concept’ for capturing the energetic constraint on of the distribution and strength of interactions.
+It is well established that consumers make more active decisions than eating items in proportion to their abundance [@stephensForagingTheory1986]. Ultimately, consumer choice is underpinned by an energetic cost-benefit framework centred around profitability and defined by traits associated with finding, catching, killing, and consuming a resource [@woottonModularTheoryTrophic2023]. Energetic constraints are invoked to construct networks in a myriad of ways [*e.g.,* **ADD DIET BREADTH MODEL 2006** @pawarDimensionalityConsumerSearch2012; @portalierMechanicsPredatorPrey2019; **NOTE Portalier creates a pairwise list, not a network** @cherifEnvironmentRescueCan2024]. In contrast to requiring a combination of foraging rules and a derived metaweb as noted above, many of these methods deliver a realised web directly. Note that we select profitability as a term to capture rules linked to optimal foraging [@pykeOptimalForagingTheory1984] and metabolic theory [@brownMetabolicTheoryEcology2004]; it is a sensible ‘umbrella concept’ for capturing the energetic constraint on of the distribution and strength of interactions.
```

Comment:
Although Portalier creates a species list *theoretically* the resulting network would align with a realised network.. My core rationale for placing the work here is that interactions are solely based on energetic feasibility and once again comes around to the giraffe eating gazelle discussion

Reply:
OK.